services/README.md

```diff
@@ -0,0 +1,42 @@
+# Buddy App GraphQL service
+
+> Service that exposes GraphQL lambda to be consumed by Buddy App front end app. Connects also to a prisma db.
+
+## Prerequisites
+
+- Please assure that you have Node.js installed on your machine.
```

Comment:
It could be useful for devs to _explicitly_ provide node.js version that matches one defined inside  `serverless.yml` file.

https://github.com/BuildIt-Poland/buddy-app/blob/ffe929d31dfa9a822c729e322ea0d8dea5660bbe/services/serverless.yml#L5

Reply:
fixed 2ed233f